service/src/main/kotlin/fi/espoo/evaka/attendance/StaffAttendanceQueries.kt

```diff
@@ -522,6 +522,34 @@ WHERE a.departed IS NULL AND a.arrived < :startOfDay AND a.group_id = g.id AND N
         .bind("startOfDay", now.atStartOfDay())
         .bind("defaultDepartureTime", defaultDepartureTime)
         .execute()
+
+    createUpdate(
+            // language=SQL
+            """
+UPDATE staff_attendance_realtime a
+SET departed = a.arrived + interval '24 hours'
+FROM daycare_group g
+JOIN daycare d ON g.daycare_id = d.id
+WHERE a.departed IS NULL AND a.arrived + INTERVAL '24 hours' <= :now AND a.group_id = g.id AND d.round_the_clock
+        """
+                .trimIndent()
+        )
+        .bind("now", now)
+        .execute()
+
+    createUpdate(
+            // language=SQL
+            """
+UPDATE staff_attendance_external a
+SET departed = a.arrived + interval '24 hours'
+FROM daycare_group g
+JOIN daycare d ON g.daycare_id = d.id
+WHERE a.departed IS NULL AND a.arrived + INTERVAL '24 hours' <= :now AND a.group_id = g.id AND d.round_the_clock
+        """
+                .trimIndent()
+        )
+        .bind("now", now)
+        .execute()
```

Comment:
Everything this function does should be documented in the wiki

Reply:
I added a wiki entry (to be merged after this PR)